jwst/extract_1d/extract.py

```diff
@@ -1313,6 +1313,13 @@ def locn_from_wcs(self, input_model, slit, targ_ra, targ_dec, verbose):
             wavelength attribute or wcs function is not defined.
         """
 
+        # WFSS data are not currently supported.
+        if input_model.meta.exposure.type in WFSS_EXPTYPES:
+            log.warning("For exposure type %s, we currently can't use "
+                        "target coordinates to get location of spectrum.",
+                        input_model.meta.exposure.type)
+            return None, None, None
```

Comment:
This seems odd, but I understand if this method is expected to return a 3-tuple. 

Would it be better maybe to assign those values to attributes instead and then return them?